lua/usip_player.lua

```diff
@@ -1180,24 +1180,27 @@ function readSip(strHbootPath, tPlugin, strTmpFolderPath, atPluginOptions, strEx
                 )
             end
 
-            tFlasherHelper.sleep_s(5)
             tLog.info("Disconnect from Plugin and reconnect again")
             -- can there be timing issues with different OS
             tPlugin:Disconnect()
-            tFlasherHelper.sleep_s(8)
+            -- wait at least 2 sec for signature verification of read sip binary
+            tFlasherHelper.sleep_s(3)
 
             while uLRetries > 0 do
                 tLog.info("try to get the Plugin again after read sip reset")
-                tPlugin = tFlasherHelper.getPlugin(strPluginName, strPluginType, atPluginOptions)
-                if tPlugin ~= nil then
+                -- tPlugin = tFlasherHelper.getPlugin(strPluginName, strPluginType, atPluginOptions)
+                fCallSuccess, tPlugin = pcall(
```

Comment:
Does getPlugin actually fail and then succeed after a retry?

Reply:
maybe with ethernet interface